security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolation.java

```diff
@@ -194,19 +272,37 @@ public double getValue() {
         return value;
     }
 
+    /**
+     * The side of the branch side where the violation occurred.
+     *
+     * @return the side of the branch side where the violation occurred.
+     */
+    public Branch.Side getBranchSide() {
```

Comment:
This must be public? This is confusing...

Reply:
This is a convenient function for the user to avoid having to cast to BranchSide himself. We could only return a ThreeSide and let the user do the cast (for a call to getTerminal for example), but then we acknowledge that the user must know if the limit violation concerns a Branch or a 3wt, so giving him the tools to interpret the side the way he want is convenient.